src/coreclr/src/jit/codegenlinear.cpp

```diff
@@ -741,15 +730,73 @@ void CodeGen::genCodeForBBlist()
 #endif // !FEATURE_EH_FUNCLETS
 
             case BBJ_NONE:
-            case BBJ_COND:
             case BBJ_SWITCH:
                 break;
 
+            case BBJ_ALWAYS:
```

Comment:
No, for 2 reasons: 
1. The loop code with switch is already complicated and loop alignment won't help in anyway.
2. The logic to check if the jump dst of switch has `BBF_LOOP_ALIGN` flag gets complicated because we have to look in `bbJumpSwt` table to scan all the blocks to check for that.